src/components/ha-expansion-panel.ts

```diff
@@ -0,0 +1,134 @@
+import {
+  css,
+  CSSResult,
+  customElement,
+  html,
+  LitElement,
+  property,
+  TemplateResult,
+} from "lit-element";
+import { fireEvent } from "../common/dom/fire_event";
+import "./ha-svg-icon";
+import { mdiChevronDown } from "@mdi/js";
+
+@customElement("ha-expansion-panel")
+class HaExpansionPanel extends LitElement {
+  @property({ type: String }) title = "";
+
+  @property({ type: Boolean }) expanded = false;
+
+  protected render(): TemplateResult {
+    return html`
+      <div
+        class="summary ${this.expanded ? "expanded" : ""}"
+        @click=${this._togglePanel}
+      >
+        <div class="summary-content">
+          <span class="summary-title">
+            ${this.title}
+          </span>
+        </div>
+        <div class="summary-icon ${this.expanded ? "expanded" : ""}">
+          <ha-svg-icon path=${mdiChevronDown}></ha-svg-icon>
+        </div>
+      </div>
+      <div class="container ${this.expanded ? "expanded" : ""}">
```

Comment:
class map. 

Maybe a parent div would be better 🤷

Reply:
done. I removed one classMap and used CSS selector